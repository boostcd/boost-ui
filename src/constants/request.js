export const REQUEST_TIMEOUT = 10000;

export const REQUEST_RESPONSE_STATUS = {
  SUCCESS: 'success',
  ERROR: 'error',
};
