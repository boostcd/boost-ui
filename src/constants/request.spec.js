import { REQUEST_RESPONSE_STATUS, REQUEST_TIMEOUT } from './request';

describe('Constants: request', () => {
  it('should have the REQUEST_TIMEOUT defined', () => {
    expect(REQUEST_TIMEOUT).toBeDefined();
    expect(typeof REQUEST_TIMEOUT).toBe('number');
  });

  it('should match the snapshot - REQUEST_RESPONSE_STATUS', () => {
    expect(REQUEST_RESPONSE_STATUS).toMatchSnapshot();
  });
});
