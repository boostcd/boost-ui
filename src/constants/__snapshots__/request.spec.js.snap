// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Constants: request should match the snapshot - REQUEST_RESPONSE_STATUS 1`] = `
Object {
  "ERROR": "error",
  "SUCCESS": "success",
}
`;
